src/constants/navigation.ts

```diff
@@ -0,0 +1,11 @@
+import ImageHeader from '@components/ImageHeader';
+import { WHITE } from '@constants/colors';
+
+import { HEADER_HEIGHT } from './dimensions';
+
+export const DEFAULT_NAV_OPTIONS = {
+  headerBackground: ImageHeader,
+  headerTintColor: WHITE,
+  headerTitleStyle: { fontWeight: 'bold' },
+  headerStyle: { height: HEADER_HEIGHT, shadowColor: 'transparent' }
```

Comment:
You forgot the header `backgroundColor` according to [Invision](https://projects.invisionapp.com/d/main#/console/10158364/217697969/inspect), use the same as the screen